client/src/components/SentimentAnalysis/SentimentAnalysis.js

```diff
@@ -116,20 +118,17 @@ const SentimentAnalysis = () => {
   };
 
   /**
-   * Generates a PNG screenshot of the current result page and
+   * Generates a JSON file of the current result page and
```

Comment:
Minor nit: it's just creating a json file of the language analysis results right?

Reply:
yeah, and the content is included as well!